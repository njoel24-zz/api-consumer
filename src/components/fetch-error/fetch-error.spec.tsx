import React from 'react';
import renderer from 'react-test-renderer';

import FetchError from './fetch-error.component';

describe('Fetch Error', () => {
	it('renders component and compare with snapshot', () => {
		const component = renderer.create(<FetchError/>);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});