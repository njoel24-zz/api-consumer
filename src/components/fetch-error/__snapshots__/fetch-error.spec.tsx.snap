// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fetch Error renders component and compare with snapshot 1`] = `
<div>
  <h4>
    Something went wrong. Most probably data is not availbale for the country selected
  </h4>
</div>
`;
